pkg/entries/api_entry.go

```diff
@@ -56,6 +56,7 @@ func readSearchOptions(r *http.Request) SearchOptions {
 		RoutingNumber: query.Get("routingNumber"),
 		TraceNumber:   query.Get("traceNumber"),
 		CreatedAfter:  query.Get("createdAfter"),
+		SubDirectory:  query.Get("subDirectory"),
```

Comment:
Can we call this path or directory? I don't think subDirectory makes sense to a user of ach-test-harness. 

```suggestion
		SubDirectory:  query.Get("subDirectory"),
```

Reply:
I think "path" is good